.github/workflows/dash.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Dash Project
+
+on: 
+  push:
+    paths:
+      - 'legacy/dash/**'
+  pull_request:
+    paths:
+      - 'legacy/dash/**'
+  
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: Cache pip
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+
+    - name: Cache PlatformIO
+      uses: actions/cache@v2
+      with:
+        path: ~/.platformio
+        key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+
+    - name: Install PlatformIO
+      run: |
+        python -m pip install --upgrade pip
+        pip install --upgrade platformio
+
+    - name: Build Dash Project
```

Comment:
Do we need to cut and paste this entire file each time? Why not add a named job for dash like you did here? Instead of redoing the PIO setup steps, perhaps we can just have one file with a named job for dash steering etc. Example:

```
- name: Build Dash Project
  run: pio run -e uno
  working-directory: legacy/dash/SIM/

- name: Build Steering
  run: pio run -e uno
  working-directory: steering/
```

Reply:
yea true - there is a way to make a composite run step that takes inputs (https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action). we could basically write a composite action that takes in the platform and working directory

Reply:
> > Any thoughts / do we care to be more specific in the project folder? perhaps `legacy/q19-dash/`?
> > Can't speak on the CI stuff Ethan brought up but once that's sorted looks good to me.
> 
> Personally don't think it matters. In my opinion "legacy" will only date back to Q19, since there isn't merit in maintaining software for hardware we don't have (i.e. the Q17/18 dash).

Could we put a README in the legacy folder explaining this then? Also, I think we may still want to have the name indicate it is q19 dash code since we may also choose to include some of the partially developed dash code we had in q20.

Reply:
Okay, updated to be `q19-dash`and added a readme.